Makefile

```diff
@@ -21,6 +21,7 @@ create_networks:
 setup:
```

Comment:
shouldn't these three just be dependent targets? Makes them parallelizable, idiomatic, etc

Reply:
Yes. After discussing with Erez we agreed to move this to a separate PR. I think it something that is also present in other makefiles.